CMakeLists.txt

```diff
@@ -169,6 +176,7 @@ set(HIP_OFFLOAD_LINK_OPTIONS "" CACHE STRING "space separated list of compiler f
 
 # =============================================================================
 # CHIP-SPV CMAKE OPTIONS PARSING
+list(APPEND CHIP_SPV_DEFINITIONS DEFAULT_WARP_SIZE=${DEFAULT_WARP_SIZE})
```

Comment:
Seems redundant as the warp size is put into the Config.hh too. I'd think the definition in the header is enough.

Reply:
It's not included everywhere, unfortunately.

Reply:
I made it installable (renamed to CHIPSPVConfig.hh to avoid file name clashes) and now included from the builtin header.